src/patternfly/components/AboutModalBox/about-modal-box.scss

```diff
@@ -219,23 +213,8 @@
     display: flex;
     align-items: center;
     justify-content: center;
-    width: var(--#{$about-modal-box}__close--c-button--Width);
-    height: var(--#{$about-modal-box}__close--c-button--Height);
     padding: 0;
     font-size: var(--#{$about-modal-box}__close--c-button--FontSize);
     color: var(--#{$about-modal-box}__close--c-button--Color);
-    background-color: var(--#{$about-modal-box}__close--c-button--BackgroundColor);
-    border-radius: var(--#{$about-modal-box}__close--c-button--BorderRadius);
-
-    &:hover {
-      --#{$about-modal-box}__close--c-button--BackgroundColor: var(--#{$about-modal-box}__close--c-button--hover--BackgroundColor);
-    }
```

Comment:
I think you can get rid of all of this custom CSS aside from the font-size. Everything else should just come from the plain button. All of this is because it is a circle shape in PF4/5.